compiler_gym/leaderboard/llvm_instcount.py

```diff
@@ -84,61 +87,69 @@
 # interacts with that environment with the goal of maximising cumulative reward.
 Policy = Callable[[LlvmEnv], None]
 
+def eval_policy(d):
```

Comment:
```suggestion
def _eval_policy(d):
```

Nitpick: use a private function so that the auto generate [sphinx docs](https://facebookresearch.github.io/CompilerGym/compiler_gym/leaderboard.html#module-compiler_gym.leaderboard.llvm_instcount) don't create an entry for it